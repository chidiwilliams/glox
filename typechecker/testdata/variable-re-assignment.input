var x: number = 20;
x = "hello";
