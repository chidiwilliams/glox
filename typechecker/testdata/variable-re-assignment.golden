error on line 2: expected 'number' type, but got 'string'
